### DISCUSSION TITLE
INDEXER

### DISCUSSION THREAD

**Discussion by harrybite** [blockscout/blockscout#9192] [General]
I am running my EVM blockchain on AWS and using Blockscout for Explorer. I have been running Blockscout since November 2023, and it has indexed 20% of blocks. Currently, there are 3.2M+ blocks, and increasing in every 6 seconds, Is there any way to speed up the indexing?
Because if it goes like this, it will take a year to index the full blockchain

**Comment by ekawibisono** [comment]
hello @harrybite can you read docs this https://docs.blockscout.com/for-developers/indexing

**Comment by harrybite** [comment]
It means it can not be sped up, then I have to wait until it indexes all blocks.
Thanks @ekawibisono

**Reply by ekawibisono** [reply]
All indexers do have to be patient, because they read everything in the JSON RPC content

**Reply by GimluCom** [reply]
@harrybite I don't know which AWS instance you use but you may be able to increase speed by increasing hardware/machine resources.
https://docs.blockscout.com/for-developers/information-and-settings/requirements/resource-requirements
